<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: #292C34;
color:white;
}</string>
  </property>
  <widget class="QLineEdit" name="chaserText">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="chaserButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Chaser</string>
   </property>
  </widget>
  <widget class="QSlider" name="chaserSlider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>60</y>
     <width>31</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="rainbowText">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="rainbowButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>270</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 1px solid white;
border-radius: 5px;
background-color: #292C34;
color: white;
}</string>
   </property>
   <property name="text">
    <string>Rainbow</string>
   </property>
  </widget>
  <widget class="QSlider" name="rainbowSlider">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>31</width>
     <height>160</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
